Gymphony/src/app/features/user-profile/pages/user-profile/user-profile.component.html

```diff
@@ -0,0 +1,31 @@
+<div *ngIf="user$ | async as user" class="flex items-center justify-center h-screen">
+    <div class="grid w-full h-screen grid-cols-3 gap-8 p-4">
+      
+      <mat-card class="h-full col-span-1">
+        <mat-card-content>
+          <app-profile-picture *ngIf="user$ | async as user" [user]="user"></app-profile-picture>
```

Comment:
You already have user variable above, it's available to you, no need to subscribe again